src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int UPPER_BOUND = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(UPPER_BOUND),
+                new ColorSupplier().getRandomColor().name());
```

Comment:
no need to create ColorSupplier each time. you can create a private property and initialize it as before the last commit. Just make that property final

Reply:
But why it wasn't fixed before last commit? there are too many opinioin which is related to the developer's self and rules. Do not think that more 1 or less 1 line is a huge mistake.

Reply:
really, not ever suggstion is better imho